Task1/CommaSeperator.js

```diff
@@ -0,0 +1,4 @@
+// 1- https://www.codewars.com/kata/56e2f59fb2ed128081001328/train/javascript
+function printArray(array){
+    return array.toString();

```

Comment:
Their is a separate method in java script that is used for this purpose.
array.join();

Look into the document link below to learn more about array.join():
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Go through the array methods in the document, and use the appropriate method